api/address.go

```diff
@@ -13,7 +13,8 @@ import (
 // Address is the interface that defines methods to manage Filecoin addresses and wallets.
 type Address interface {
```

Comment:
Aside: this is a confusing name for an interface that contains multiple addresses.

Reply:
Agreed, this bit is even more confusing: https://github.com/filecoin-project/go-filecoin/issues/1693.